helpers/helpers.go

```diff
@@ -288,6 +290,50 @@ func GetAuth(path string) (*Config, error) {
 	return nil, errors.New("failed to read the configuration file")
 }
 
+func GetPublicKey() (string, error) {
```

Comment:
It would be useful to add specific tests in the `helpers_test.go` suite for all new functions.

Reply:
I meant `GetPublicKey() (string, error)` etc, not the ones imported :-)

Reply:
added some